src/atecc/atecc.c

```diff
@@ -34,7 +34,6 @@ typedef enum {
     ATECC_SLOT_ROLLKEY = 3,
     ATECC_SLOT_KDF = 4,
     ATECC_SLOT_ATTESTATION = 5,
-    ATECC_SLOT_ECC_UNSAFE_SIGN = 6,
```

Comment:
Maybe we should keep this enumeration around, could rename it to DEPRECATED/RESERVED or something similar. So that we don't forget that it has been used.